<launch>
  <param name= "/loopRate" type= "double" value= "250"/>

  <param name= "/filePath" type= "string" value= "/home/mxr/catkin_laikago/calculated_crawl_two.txt"/>

  <node name="kinematics_control_node" pkg="quadruped_model" type="kinematics_control_node" output="screen">
  </node>

</launch>
